cmd/snap/cmd_warnings_test.go

```diff
@@ -102,28 +102,44 @@ func (s *warningSuite) TestNoFurtherWarnings(c *check.C) {
 func (s *warningSuite) TestWarnings(c *check.C) {
 	s.RedirectClientToTestServer(mkWarningsFakeHandler(c, twoWarnings))
 
-	rest, err := snap.Parser(snap.Client()).ParseArgs([]string{"warnings", "--abs-time"})
+	rest, err := snap.Parser(snap.Client()).ParseArgs([]string{"warnings", "--abs-time", "--unicode=never"})
 	c.Assert(err, check.IsNil)
 	c.Check(rest, check.HasLen, 0)
 	c.Check(s.Stderr(), check.Equals, "")
 	c.Check(s.Stdout(), check.Equals, `
-Last occurrence       Warning
-2018-09-19T12:41:18Z  hello world number one
-2018-09-19T12:44:19Z  hello world number two
+Last occurrence:  2018-09-19T12:41:18Z
+Warning: |
+  hello world number one
```

Comment:
do we need a test where one of these actually wraps around ?

Reply:
it's using `printDescr` which already has coverage for that.
OTOH we could change it to use `printSummary`, which has less coverage, and then I'd say yes let's add something.

Reply:
Ok, problem: we can't use printSummary because it doesn't  know how much padding is going to get added (in `info`, the keys aren't translated so it's easier to know).